.github/workflows/Ontology_Mapper.yml

```diff
@@ -0,0 +1,71 @@
+name: Ontology Mapper 
+
+on: push
+
+env:
+  POETRY_VERSION: '1.6.1'
+
+jobs:
+  run-script:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
+    - name: Setup Python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.9"
+
+    - name: Set up poetry
+      uses: abatilo/actions-poetry@v2.1.5
+      with:
+        poetry-version: "${{ env.POETRY_VERSION}}"
+
+    - name: Install Dependencies with Poetry
+      run: |
+        poetry source add --priority=primary PyPI
+        poetry source add --priority=supplemental fury https://pypi.fury.io/mapped/
+        poetry config http-basic.fury ${{ secrets.FURY_READ_TOKEN_PUBLIC }} NOPASS
+        poetry lock --no-update
+        poetry install --only main
+
+    - name: Generate Mappings
+      run: poetry run python scripts/generate_mappings.py
+    
+    - name: Sort Mapped Mappings
+      run: poetry run python scripts/sort_mappings.py Ontologies.Mappings/src/Mappings/v1/Mapped/willow_v1_dtdlv2_mapped.json
+
+    - name: Sort Willow Mappings
+      run: poetry run python scripts/sort_mappings.py Ontologies.Mappings/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json
+
+    - name: Upload Mapped Mappings Artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: mapped-mappings
+        path: Ontologies.Mappings/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json
+
+    - name: Upload Willow Mappings Artifact
```

Comment:
Create artifact from a single step

Reply:
@jbkoh Can you clarify what you mean by this? From my understanding, I am already creating the artifact in a single step